modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-3-expired-certs.adoc
+
+[id="dr-scenario-3-recovering-expired-certs_{context}"]
+= Recovering from expired control plane certificates
+
+Follow this procedure to recover from a situation where your control plane certificates have expired.
+
+.Prerequisites
+
+* SSH access to master hosts.
+
+.Procedure
+
+. Access a master host with an expired certificate as the root user.
+
+. Obtain the `cluster-kube-apiserver-operator` image information for a release.
++
+----
+# KAO_IMAGE=$( oc adm release info --registry-config='/var/lib/kubelet/config.json' <release_image> --image-for=cluster-kube-apiserver-operator ) <1>
+----
++
+<1> An example value for `<release_image>` is `quay.io/openshift-release-dev/ocp-release:4.1.0-rc.0`.
```

Comment:
might be best to leave the rc part out of it

Reply:
@tnozicka if i take out the "-rc", should I also take out the ".0" after it? So the version is just "4.1.0"?